print("main fault has occurred")